lib/src/metadata.dart

```diff
@@ -39,7 +39,7 @@ class SymbolMetadata {
 /// within the protobuf schema for scip
 SymbolMetadata getSymbolMetadata(
   Element element,
-  AstNode node,
+  int offset,
```

Comment:
the specific `node` we pass in here isn't always the offset of the specific entity. For example, some elements are indexed by some analyzed `element` instead of the ast node at the specific depth

For this reason, just pass the offset as an int, so the errors can be correctly calculated where necessary